### PR TITLE
Adding doubleml

### PR DESCRIPTION
**Pull request by MalteKurz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/doubleml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  summary: 'Double Machine Learning in Python'
+  description: |
+    The Python package DoubleML provides an implementation of the double / debiased machine learning framework of
+    Chernozhukov et al. (2018). It is built on top of scikit-learn (Pedregosa et al., 2011).
+  doc_url: https://docs.doubleml.org/stable/
+  dev_url: https://github.com/DoubleML/doubleml-for-py
+
+extra:
+  recipe-maintainers:
+    - MalteKurz
```
Comment (MalteKurz) [inline recipes/doubleml/meta.yaml:55]: I confirm that I am willing to be listed as maintainer of this recipe

File: recipes/doubleml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/D/DoubleML/DoubleML-{{ version }}.tar.gz
+  sha256: 32ef1d20cd44cf628313a6a27c6ab00041414a531fb0a5bec3e7c77c2dfde875
+
+build:
+  number: 0
+  skip: true  # [py<36]
```
Comment (synapticarbors) [inline recipes/doubleml/meta.yaml:14]: Remove `skip` and make this `noarch: python` and we'll restrict the python version below:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/doubleml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/doubleml/meta.yaml:18]: remove `build` section under `requirements`

File: recipes/doubleml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - setuptools
+  host:
+    - python
+    - pip
+    - numpy
```
Comment (synapticarbors) [inline recipes/doubleml/meta.yaml:23]: I don't think you need `numpy` under `host`

File: recipes/doubleml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+  host:
+    - python
```
Comment (synapticarbors) [inline recipes/doubleml/meta.yaml:21]: Change `python` to `python >=3.6` here and under `run`.

**Comment by MalteKurz** [comment]
Thank you for the review @synapticarbors. I tried to address all raised points appropriately.
The PR is ready for review again.

**Pull request merged**